x-pack/plugins/monitoring/server/routes/api/v1/elasticsearch_settings/check/internal_monitoring.ts

```diff
@@ -62,7 +83,12 @@ export function internalMonitoringCheckRoute(server: unknown, npRoute: RouteDepe
           mb_indices: 0,
         };
 
-        const { esIndexPattern, kbnIndexPattern, lsIndexPattern } = getIndexPatterns(server);
+        const config = server.config();
+        const ccs = '*';
```

Comment:
This is another problem here. `ccs` should only equal `*` if we're doing this API call from the cluster listing page, otherwise, the client needs to send this up. 

What is happening is I have two clusters and one cluster is internal and one is MB monitored. Whenever I refresh any page on the MB monitored one, I see the toast